segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperServiceRunner.java

```diff
@@ -68,11 +69,35 @@ public void close() throws Exception {
     //region BookKeeper operations
 
     /**
+     * Suspends the BookieService with the given index (does not stop it).
```

Comment:
What does it mean to suspend the bookie service? I understand intuitively, but I´m not sure what it is actually doing.